src/components/Layout/Hero/HeroStyles.ts

```diff
@@ -7,6 +7,21 @@ const StyledHero = styled.h2`
   font-family: "Rubik Moonrocks", cursive;
   text-align: center;
   font-weight: normal;
+
+  @media (max-width: 920px) {
+    font-size: 3rem;
+    padding-top: 1rem;
+    padding-bottom: 0.5rem;
+  }
+
+  @media (max-width: 710px) {
+    font-size: 2.5rem;
+    padding-bottom: 0rem;
+  }
+
+  @media (max-width: 540px) {
+    font-size: 2rem;
+  }
 `;
```

Comment:
it's better when all the media queries are consistent and coming from your theme